### PR TITLE
feat(learn): add backend solution url to help post

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

With this, when a Camper uses the Ask for Help button on a challenge, the pre-populated post now includes a link to their solution url on backend challenges.

This just helps us on the forum, when Campers forget to include a link to the code they are trying to submit.

In order for this to work, the tests have to be run at least once with the given URL. (They do not need to pass)

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]
LGTM. @RandellDawson what do you think of this change?

**Comment by RandellDawson** [comment]
@raisedadead I corresponded with @ShaunSHamilton on chat and he is going to be making a change.  Once that is done, I think the PR will be good to go.

**Comment by ShaunSHamilton** [comment]
The latest changes incorporate Randell's suggestion to look like this:
image

File: client/src/templates/Challenges/redux/create-question-epic.js [hunk 1]
```diff
@@ -52,11 +54,11 @@ function createQuestionEpic(action$, state$, { window }) {
         `**Tell us what's happening:**
         \n\n
         ${
-          projectFormValues
+          projectFormValues.length > 0
```
Comment (RandellDawson) [inline client/src/templates/Challenges/redux/create-question-epic.js:57]: The following should suffice?
```suggestion
          projectFormValues.length
```

**Pull request merged**